draft-wang-ppm-differential-privacy.md

```diff
@@ -578,41 +564,41 @@ class DpMechanism:
 
 > TODO: Specify a Gaussian sampler from Algorithm 3 of {{CKS20}} (#10).
 
-## Binary Randomized Response {#rr}
+## Symmetric RAPPOR {#symmetric-rappor}
 
-> KT: This is technically symmetric RAPPOR, so may be more appropriate to call it that?
-> Or at least cite the RAPPOR paper as the first proposing it.
+This section describes symmetric RAPPOR that was first proposed in {{EPK14}},
```

Comment:
@kunal-talwar Let me know if this is appropriate now, or if you have other suggestions, e.g., we should instantiate symmetric RAPPOR from the original paper.